src/elements/fields/ButtonGroupField.jsx

```diff
@@ -1,17 +1,25 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import { imgMaxSizeStyles } from '../styles';
 
 function ButtonGroupField({
   element,
   applyStyles,
   fieldLabel,
-  fieldVal = null,
+  fieldVal = [],
```

Comment:
it might be good to leave fieldVal by default to be null since a default array param will trigger the useMemo every render

Reply:
ty for catching